Gems/Atom/RPI/Code/Source/RPI.Edit/Shader/ShaderSourceData.cpp

```diff
@@ -20,12 +20,14 @@ namespace AZ
             if (auto* serializeContext = azrtti_cast<SerializeContext*>(context))
             {
                 serializeContext->Class<ShaderSourceData>()
-                    ->Version(6) // Introduction of "AddBuildArguments" & "RemoveBuildArguments"
+                    ->Version(7) // Rework of the blend state options
                     ->Field("Source", &ShaderSourceData::m_source)
                     ->Field("DrawList", &ShaderSourceData::m_drawListName)
                     ->Field("DepthStencilState", &ShaderSourceData::m_depthStencilState)
                     ->Field("RasterState", &ShaderSourceData::m_rasterState)
                     ->Field("BlendState", &ShaderSourceData::m_blendState)
+                    ->Field("GlobalTargetBlendState", &ShaderSourceData::m_globalTargetBlendState)
+                    ->Field("TargetBlendStates", &ShaderSourceData::m_targetBlendStates)
```

Comment:
I was going to say that https://www.o3de.org/docs/atom-guide/look-dev/shaders/shader-file-spec/ needs to be updated, but actually that page doesn't even mention BlendState. Anyway, I still recommend updating the page after merging these changes. 

See https://github.com/o3de/o3de.org

Reply:
I was not aware of this page, I'll prepare something for it.